project.sh

```diff
@@ -570,7 +570,7 @@ local-clean(){
 			if is-java;then
 				mvn --update-snapshots dependency:go-offline -Dverbose
 				abort-on-error
-				if ! is-ci;then
+				if ! is-ci && [[ -z "$JESI_DISABLE_MVN_SOURCE_DOWNLOAD" ]];then
```

Comment:
30+ minutes to just do deps... Keeps re-downloading sources and docs it already has.